azurelinuxagent/ga/exthandlers.py

```diff
@@ -316,10 +316,13 @@ def handle_ext_handlers(self, etag=None):
             return
 
         if conf.get_enable_overprovisioning():
-            artifacts_profile = self.protocol.get_artifacts_profile()
-            if artifacts_profile and artifacts_profile.is_on_hold():
-                logger.info("Extension handling is on hold")
-                return
+            if not self.protocol.supports_overprovisioning():
+                logger.warn("Overprovisioning is enabled but protocol does not support it.")
```

Comment:
Sorry, my bad! I just realized this is in a loop that executes too frequently so the logs on Azure Stack would be too verbose. Can we change warn() to verbose()?

Reply:
Thank you, and sorry for the extra change!